pints/tests/test_slice_stepout.py

```diff
@@ -0,0 +1,350 @@
+# -*- coding: utf-8 -*-
+#
+# Tests the basic methods of the Slice Sampling routine.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+
+import unittest
+import numpy as np
+
+import pints
+import pints.toy
+
+debug = False
+
+
+class TestSliceStepout(unittest.TestCase):
+    """
+    Tests the basic methods of the Slice Sampling with Stepout routine.
+
+    Please refer to the _slice_stepout.py script in ..\_mcmc\_slice_stepout.py
```

Comment:
This is a nice touch!
Unfortunately, it could go out of date if we refactor the code at some point, so maybe best to remove it?